cypress/e2e/test/login-verification.cy.js

```diff
@@ -0,0 +1,34 @@
+const email = "hellouser@noroff.no";
+const password = "12345678";
+const wrongEmail = "hellouser0@noroff.no";
+const wrongPassword = "1234567890";
+
+describe("Login test", () => {
+  it("should not allow user to login with wrong password", () => {
+    cy.visit("/");
+    cy.wait(1000);
+    cy.get("#registerForm button[type=button]").contains("Login").click();
+    cy.wait(1000);
+    cy.get("#loginEmail").type(email, "{enter}");
+    cy.get("#loginPassword").type(wrongPassword);
+    cy.get("#loginForm button[type=submit]").contains("Login").click();
```

Comment:
What happens next? There should be something to check that it didn't go through, like an alert, error, etc.